youths/tests/test_graphql_queries.py

```diff
@@ -25,15 +29,29 @@ def test_anon_user_query_should_fail(rf, youth_profile, anon_user_gql_client):
         }
         """
     )
-    query = t.substitute(id=uuid.uuid4())
+    if use_proper_profile_id:
+        query = t.substitute(id=to_global_id(type="ProfileNode", id=uuid.uuid4()))
+    else:
+        query = t.substitute(id=to_global_id(type="ProfileNode", id=youth_profile.pk))
```

Comment:
not that it matters for the test, but should it not be the other way around? 
```python
if use_proper_profile_id:
    query = t.substitute(id=to_global_id(type="ProfileNode", id=youth_profile.pk))
else:
    query = t.substitute(id=to_global_id(type="ProfileNode", id=uuid.uuid4()))
```

Reply:
It should. :sweat: